bullet_train-themes-light/config/locales/themes/navigation.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  themes:
+    navigation:
```

Comment:
I'm actually fine if we move all of these to `bullet_train-themes` since they're not scoped under `light`, but if we keep it here maybe we can add `light:` to the hierarchy, or just put these locales inside `bullet_train-themes-light/config/locales/themes/light.yml`.

Reply:
I'm thinking the following:

* I'd keep the locale files throughout each package, and like you did @3pns to have the translation string used in that package in a locale file (`navigation.en.yml`, `fields.en.yml`, etc) within that package. So disagreeing with @gazayas a little bit here. And my argument follows the next point...
* I'd remove the `themes` grouping entirely.

I feel these translation files aren't for a `themes` concept. They're just UI strings in partials that are bundled with the theme packages. A `themes.en.yml` would be great if BT had a UI for picking the theme settings (e.g. to change the accent color to a custom color just for your account). But the theme includes strings used for navigation (and that goes in a `navigation.en.yml` in the package that defines it, etc), for fields, for specific private gems (action_models), and so on...

All these strings are also overridable in the main app. It's easy to create a `navigation.en.yml` in the main app, and override just the `close_app_menu` entry. We don't know that it was in a theme-specific package, only that I'm modifying that UI string in my own app.

Reply:
That means that:
* in the case of `light.yml`, its strings would be in a `conversations.en.yml`
* in the case of `time_ago`, that would simply be in a `en.yml` at the base of the `en/` directory in that package.

Reply:
I see where you're coming from about using `themes`. Like I said I'm open either way, I think I just prefer it all to be unified so it's all grouped together in a meaningful way, so I agree with the statement you made here:

> in the case of light.yml, its strings would be in a conversations.en.yml

If we do this, we don't have to worry about using `light` in our keys. Why don't we do that so this one can be ready to merge?

@3pns Sorry to have you make all of these changes! Can we remove `themes` and `light` from the keys you wrote up and just use general files like `conversations.en.yml`? If we do that then I'll go ahead and approve the changes.